Test-SecretVault fails due to missing hyphen

When running Test-SecretVault for a new vault signin fails with the following error: [ERROR] 2021/02/14 15:17:19 unknown shorthand flag: 'a' in -aw

This appears to be due to a missing hyphen on line 34 of SecretManagement.1Password.Extension.psm1

--- a/SecretManagement.1Password.Extension/SecretManagement.1Password.Extension.psm1
+++ b/SecretManagement.1Password.Extension/SecretManagement.1Password.Extension.psm1
@@ -26,16 +26,16 @@ function Test-SecretVault {
 
     if ($null -eq $vaults) {
         if ( $null -eq [System.Environment]::GetEnvironmentVariable("OP_SESSION_$accountName") ) {
-            Write-Verbose "Attemp login with shorthand and grab session token"
+            Write-Verbose "Attempt login with shorthand and grab session token"
             $token = & op signin $accountName --raw
 
             if ( $null -eq $token ) {
-                Write-Verbose "Attemp login with all parameters"
-                $token = & op signin $accountName $emailAddress $secretKey -raw
+                Write-Verbose "Attempt login with all parameters"
+                $token = & op signin $accountName $emailAddress $secretKey --raw
             }
         }
         else {
-            Write-Verbose "Attemp login with shorthand and grab session token"
+            Write-Verbose "Attempt login with shorthand and grab session token"
             & op signin $accountName
         }
 

